packages/cli/generators/relation/templates/controller-relation-template-has-many.ts.ejs

```diff
@@ -55,7 +55,13 @@ export class <%= controllerClassName %> {
   })
   async create(
     @param.path.<%= sourceModelPrimaryKeyType %>('id') id: typeof <%= sourceModelClassName %>.prototype.<%= sourceModelPrimaryKey %>,
-    @requestBody() <%= targetModelRequestBody %>: <%= targetModelClassName %>,
+    @requestBody({
```

Comment:
The id field is not always called `id`, I think we need to get a model's id name in the template.

Reply:
Good point. AFAICT, `lb4 model` does allow users to create a primary key property with a different name than `id`.

Example prompts & answers to use `pk` instead of `id`:

```
Model Product will be created in src/models/product.model.ts

Let's add a property to Product
Enter an empty property name when done

? Enter the property name: pk
? Property type: string
? Is pk the ID property? Yes
? Is it required?: No
? Default value [leave blank for none]:
```

I agree with @jannyHou that we should honor custom PK name, there should be a new test to verify. If this is too much work then I can live with adding support for custom PK names in a follow-up pull request.

Reply:
> If this is too much work then I can live with adding support for custom PK names in a follow-up pull request.

Yeah forgot to say, feel free to address it in another PR :)

Reply:
Will do in a separate PR 👍